schema_based_on_diagram.sql

```diff
@@ -0,0 +1,55 @@
+CREATE DATABASE clinic;
+
+-- Create patients table
+CREATE TABLE patients (
+  id INT PRIMARY KEY,
+  date DATE,
+  name VARCHAR(255),
+  date_of_birth DATE
+);
+
+CREATE TABLE medical_histories (
+  id INT PRIMARY KEY, 
+  admitted_at TIMESTAMP,
+  patient_id INT,
+  status VARCHAR(255),
+  FOREIGN KEY (patient_id) REFERENCES patients(id)
+);
+
+CREATE TABLE treatments (
+  id INT PRIMARY KEY, 
+  type VARCHAR(255),
+  name VARCHAR(255)
+);
+
+CREATE TABLE invoices (
+  id INT PRIMARY KEY,
+  total_amount DECIMAL(10,2),
+    generated_at TIMESTAMP,
+  payed_at TIMESTAMP,
+  medical_history_id INT,
+  FOREIGN KEY (medical_history_id) REFERENCES medical_histories(id)
+);
+
+CREATE TABLE invoice_items (
+  id INT PRIMARY KEY,
+  unit_price DECIMAL(10,2),
+   quantity INT,
+  total_price DECIMAL(10,2),
+  invoice_id INT,
+  treatment_id INT,
+  FOREIGN KEY (invoice_id) REFERENCES invoices(id),
+  FOREIGN KEY (treatment_id) REFERENCES treatments(id)
+);
+
+CREATE TABLE treatments_histories (
+  treatment_id INT PRIMARY KEY REFERENCES treatments(id),
+  medical_history_id INT PRIMARY KEY REFERENCES medical_histories(id)
+);
+
+ALTER TABLE treatments_histories ADD CONSTRAINT fk_treatment_id FOREIGN KEY (treatment_id) REFERENCES treatments(id);
+ALTER TABLE treatments_histories ADD CONSTRAINT fk_medical_history_id FOREIGN KEY (medical_history_id) REFERENCES medical_histories(id);
+
+CREATE INDEX idx_treatment_id ON treatments_histories(treatment_id);
+CREATE INDEX idx_medical_history_id ON treatments_histories(medical_history_id);
+
```

Comment:
Great job team 🤝
- You have done well creating indexes for the `treatment_id`, and `medical_history_id`. Kindly create indexes for all Foreign Keys.
![indexes](https://github.com/Izebiafe/database_from_diagram/assets/63917725/ae27660e-14e2-4dd4-b707-09da4f541b01)